docs/how/search.md

```diff
@@ -1,46 +1,83 @@
 # Search Guide
 
-## Introduction 
+## Introduction
 
 The search bar is one of the means of finding data in Datahub. In this document, we discuss more effective ways of finding information beyond doing a standard keyword search. This is because keyword searches can return results from almost any part of an entity.
 
+### Advanced Filters
+
+Using the Advanced Filter view, you can apply more complex filters. To get there, click 'Advanced' in the top right of the filter panel:
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/advanced_search/click_to_advanced_search_view.png"/>
+</p>
+
+#### Adding an Advanced Filter
+
+Currently, Advanced Filters support filtering by Column name, Container, Domain, Description (entity or column level), Tag & Glossary Term (entity or column level), Owner, Entity Type, Subtype, Environment and soft-deleted status.
```

Comment:
should the `&` between Tag & Glossary Term just be a comma?

Reply:
yes good call